internal/lib/lib.go

```diff
@@ -1079,7 +1079,8 @@ func GetClusterIDSplit() string {
 	if clusterID != "" {
 		clusterName := strings.Split(clusterID, ":")
 		if len(clusterName) > 1 {
-			return clusterName[0]
+			// Include first 5 characters to add more uniqueness to cluster name
+			return clusterName[0] + "-" + clusterName[1][:5]
```

Comment:
What is impact of this on upgrade scenario?

Reply:
Addressed the upgrade impact by keeping the clustername consistent for non VPC based deployments. VPC deployments would only be Greenfield.